src/resources/css/tribe-common-admin.css

```diff
@@ -148,6 +148,8 @@ table.plugins .tribe-plugin-update-message a {
 .tribe-settings-form .tribe-settings-form-wrap h3 ~ h3 {
 	margin-top: 2.25em;
 }
+.tribe-settings-form .tribe-settings-form-wrap .tribe-sysinfo-optin-msg,
```

Comment:
Unless handled outside of this PR, remember that we want to update the `.pcss` file first then compile to minified and unminified CSS from there (not seeing a change to `tribe-common-admin.pcss` though).